### ISSUE TITLE
How to download transcriptome indices?

### ISSUE THREAD

**Issue by arielsvn** [AlexsLemonade/refinebio#1794]
### Context

Someone needed a transcriptome index.

### Problem or idea

We have an endpoint that should provide this information, but the download `s3_url` is empty for all results.

https://api.refine.bio/v1/transcriptome_indices/

The [docs](http://docs.refine.bio/en/latest/main_text.html#transcriptome-index) also mention people should be able to get them from here.

Looks like the organism indices don't have any `s3_url` in the DB either.

```sql
data_refinery=> select count(*) from organism_index where s3_url!='';
 count 
-------
     0
(1 row)
```

### Solution or next step

We should investigate why these values are not being populated.

**Comment by arielsvn** [comment]
Looks like the `s3_url` is not being set, and instead, they are associated with a `ComputationalResult` that is also associated with a `ComputedFile` that gets uploaded to S3

We should remove the `s3_url` form `OrganismIndex` and update the serializer to return the proper URL from the computed file.

**Comment by davidsmejia** [comment]
closed via #2012
